boot-speed/clouds/ec2_cleanup.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+"""Clean up all running bootspeed-* EC2 instances.
+
+Copyright 2018-2019 Canonical Ltd.
```

Comment:
I assume this is 2018-2019 because we're importing this script in to this repo from elsewhere?

Reply:
Yes, I took

https://github.com/CanonicalLtd/server-test-scripts/blob/master/cloud-init/ec2_cleanup.py

in this same repo and adapted it.